source/plugin/placeholders/creatingtokens.rst

```diff
@@ -39,9 +42,154 @@ This is the minimum as specified by Sponge. If another plugin replaces the :java
 If your placeholder is unable to provide text because there is insufficient information, the placeholder should return
 and empty :javadoc:`Text` and not throw an exception.
 
+Example: Default World Name PlaceholderParser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This :javadoc:`PlaceholderParser` attempts to get the default world's name, returning an empty :javadoc:`Text` if it
+cannot be found.
+
+.. code-block:: java
+
+    public class DefaultWorldNameParser implements PlaceholderParser {
+        
+        @Override
+        public String getId() {
+            return "spongedocs:defaultworld"
+        }
+
+        @Override
+        public String getName() {
+            return "Default World Placeholder"
+        }
+
+        @Override
+        public Text parse(PlaceholderText placeholderText) {
+            return Sponge.getServer()
+                .getDefaultWorld()
+                .map(x -> x.getWorldName())
+                .orElse(Text.EMPTY);
+        }
+    }
+
+Example: Player Location PlaceholderParser
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This :javadoc:`PlaceholderParser` attempts to get the player's location in the world. If used without a 
+:javadoc:`Player` as the associated receiver, returns an empty :javadoc:`Text`.
+
+.. code-block:: java
+
+    public class PlayerBalancePlaceholder implements PlaceholderParser {
```

Comment:
Just noticed this, but the example class here seems to have met the fatal copy/paste. Looks like this one should be focused on player location.

Reply:
That's from an old commit!